cmd/root.go

```
@@ -455,17 +127,17 @@ func compareIssues(ghClient github.Client, jiraClient jira.Client) error {
 	for _, ghIssue := range ghIssues {
 		found := false
 		for _, jIssue := range jiraIssues {
-			id, _ := jIssue.Fields.Unknowns.Int(fmt.Sprintf("customfield_%s", ghIDFieldID))
+			id, _ := jIssue.Fields.Unknowns.Int(config.GetFieldKey(lib.GitHubID))
```

is this suppressing an error?